packages/lesswrong/components/votes/AgreementVoteAxis.tsx

```diff
@@ -4,14 +4,15 @@ import { useCurrentUser } from '../common/withUser';
 import { voteButtonsDisabledForUser } from '../../lib/collections/users/helpers';
 import { VotingProps } from './votingProps';
 import { isFriendlyUI } from '../../themes/forumTheme';
+import { defineStyles, useStyles } from '../hooks/useStyles';
+import classNames from 'classnames';
 
-const styles = (theme: ThemeType) => ({
+const styles = defineStyles('AgreementVoteAxis', (theme: ThemeType) => ({
   root: {
   },
   agreementSection: {
     display: "inline-block",
     fontSize: 25,
-    marginLeft: 8,
```

Comment:
I'm not sure what changes these imply for the agreement vote axis outside of the new feed, but by default I'd guess we don't want them?

Reply:
Oh I see, some of the changes are just splitting out bits of `agreementScore` between `agreementScoreSmall` and `agreementScoreLarge`.  But not all of them (like this one, and the new default is 7 px side margin rather than 3?)

Reply:
I want to use the buttons independently and this applying a still here for the larger context that I'm not using, so I moved the style to the larger context (NamesAttachedReactions....)